.htaccess

```diff
@@ -156,14 +156,67 @@
 
 	# Anything in sapphire/en/<version>/contributing is redirected to trunk, the info in there is usually not version specific,
 	# and only updated on master (e.g. contribution and coding guidelines)
-	#RewriteRule ^framework/en/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
-	#RewriteRule ^framework/en/[\d.]*/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
-
+	# RewriteRule ^framework/en/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
+	# RewriteRule ^framework/en/[\d.]*/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
+
+
+	#### Issue 50 : Top 50 404 Errors ####
+	RewriteRule ^changelog/?$ /en/changelogs [R=301,L]
+	RewriteRule ^installation-into-subversion/?$ /en/getting_started/composer [R=301,L]
+	# check target with Cam
+	RewriteRule ^modules:rssaggregator/?$ http://addons.silverstripe.org/add-ons/silverstripe/versionfeed [R=301,L]
```

Comment:
Make sure when you resolve these you dont make another pull request. Just update this one.